build_test.go

```diff
@@ -2191,6 +2191,44 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 		})
+
+		when("gid option", func() {
+			when("gid is equal to zero", func() {
+				it("gid is passthroughs to lifecycle", func() {
+					h.AssertNil(t, subject.Build(context.TODO(), BuildOptions{
+						Workspace:       "app",
+						Builder:         defaultBuilderName,
+						Image:           "example.com/some/repo:tag",
+						GroupID:         0,
+					}))
+					h.AssertEq(t, fakeLifecycle.Opts.GID, 0)
+				})
+			})
+
+			when("gid is less than zero", func() {
+				it("gid is passthroughs to lifecycle", func() {
+					h.AssertNil(t, subject.Build(context.TODO(), BuildOptions{
+						Workspace:       "app",
+						Builder:         defaultBuilderName,
+						Image:           "example.com/some/repo:tag",
+						GroupID:         -1,
+					}))
+					h.AssertEq(t, fakeLifecycle.Opts.GID, -1)
+				})
+			})
+
+			when("gid is greater than zero", func() {
```

Comment:
I kinda feel like this test alone is sufficient, given that there's no actual GID manipulating logic in the current implementation for `build.go`.  In general, testing a lot is something I'm a big fan of, but I don't feel like there's so much value added by the <=0 tests. What do you think?

Reply:
@dfreilich, yeah after removing the boolean attribute I also think these tests are not adding anything valuable. I will remove them